fl4health/clients/basic_client.py

```diff
@@ -629,10 +652,20 @@ def predict(self, input: torch.Tensor) -> Tuple[Dict[str, torch.Tensor], Dict[st
             All predictions included in dictionary will be used to compute metrics.
 
         Raises:
+            TypeError: Occurs when something other than a tensor or dict of tensors is passed in to the model's
+            forward method.
             ValueError: Occurs when something other than a tensor or dict of tensors is returned by the model
             forward.
         """
-        output = self.model(input)
+        if isinstance(input, torch.Tensor):
+            output = self.model(input)
+        elif isinstance(input, dict):
+            # If input is a dictionary, then we unpack it before computing the forward pass.
+            # Note that this assumes the keys of the input match (exactly) the keyword args
+            # of self.model.forward().
+            output = self.model(**input)
+        else:
+            raise TypeError(""""input" must be of type torch.Tensor or Dict[str, torch.Tensor].""")
```

Comment:
I don't think the triple quotes are necessary here. I think you can do `"\"input\""`, but I'm not 100% sure. Normally triple quotes are reserved for line allowing line breaks into the string.